pkg/security/podsecuritypolicy/provider.go

```diff
@@ -111,7 +111,7 @@ func (s *simpleProvider) MutatePod(pod *api.Pod) error {
 	}
 
 	var retErr error
-	podutil.VisitContainers(&pod.Spec, func(c *api.Container) bool {
+	podutil.VisitContainers(&pod.Spec, podutil.DefaultContainers, func(c *api.Container, containerType podutil.ContainerType) bool {
```

Comment:
if this is an update, and the existing persisted pod has ephemeral containers, then the security policy should protect the ephemeral containers on this pod as well. if we don't want to plumb and look at the existing pod, then just visiting all containers is safer than skipping feature-disabled ones